### PR TITLE
Overhaul of Wii/GameCube game dumping guides

### PR DESCRIPTION
**Pull request by grunt22fox**
Adds images to sections, adds Wii Backup Fusion instructions, general revisions throughout the guides.

### CODE REVIEW

File: _pages/en_US/wii-backups.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+    ![](/images/desktop-apps/WBM/dumpedpartfiles.png)
+
+1. In this folder, click the address bar in Windows Explorer and copy its PATH.
+
+    ![](/images/desktop-apps/WBM/copypath.png)
+
+1. Open a Command Prompt window, and switch to the drive that the file is on (for example `B:`) and then switch to the PATH you copied earlier using `cd [PATH]`.
+
+    ![](/images/desktop-apps/WBM/terminalcd.png)
```
Comment (Naim2000) [inline _pages/en_US/wii-backups.md:37]: You can just type `cmd` in the address bar and it opens there by the way. And i think step 3 is going to confuse some people anyways, you just copied the ISO files to your computer but this makes it sound like you should switch to your external drive

File: _pages/en_US/wiibackupmanager.md
Comment (Naim2000) [inline _pages/en_US/wiibackupmanager.md:1]: Redirect? Or do we not want those at all
Same for gcbackupmanager

File: _pages/en_US/dump-games.md [hunk 2]
```diff
@@ -25,96 +22,71 @@ If you are dumping one of the 13 games on [this list](https://wiki.dolphin-emu.o
...
 
-+ A Windows/macOS/Linux PC with an internet connection
-+ [DVD Dump Tool](/assets/files/DVDDumpTool.zip)
+    ![](/images/homebrew/CleanRip/insertdisc.png)
 
-#### Instructions
+### Dumping a Wii Disc
 
-Download speeds will be slow due to limitations in the network hardware of the Wii, but it remains a valid solution if you are willing to be patient.
+If you are dumping one of the 13 games on [this list](https://wiki.dolphin-emu.org/index.php?title=Category:Dual_Layer_Disc_games), set `Dual Layer` in Step 1 to `Yes`.
```
Comment (Naim2000) [inline _pages/en_US/dump-games.md:55]: Cleanrip updated to v2.1.2 and now it has an auto option for dual layer discs (:D)
Also asks if your Wii disc is an unlicensed datel disc just like GC

**Review by Naim2000 — COMMENTED** [review]
~~Also you broke my dolphin images on the transfer-saves page :v~~ (I need to revise that thing as a whole)

File: _pages/en_US/wii-backups.md (hunk 1 above)
Comment (grunt22fox) [reply _pages/en_US/wii-backups.md]: Resolved with the latest commit

File: _pages/en_US/wiibackupmanager.md
Comment (grunt22fox) [reply _pages/en_US/wiibackupmanager.md:1]: Github pages doesn't like redirects apparently, so I think we just keep it as-is

File: _pages/en_US/dump-games.md (hunk 2 above)
Comment (grunt22fox) [reply _pages/en_US/dump-games.md:55]: Didn't update the screenshots to v2.1.2, but I did clarify that this was the case and that the version had an auto-detect option.

File: _pages/en_US/dump-games.md [hunk 3]
```diff
@@ -25,96 +22,75 @@ If you are dumping one of the 13 games on [this list](https://wiki.dolphin-emu.o
...
 
-To organize the games on your drive properly, you can use [Wii Backup Manager](wiibackupmanager).
-{: .notice--info}
+    ![](/images/homebrew/CleanRip/gcprogress.png)
+
+1. When the dump finishes, you should see a screen with the MD5 checksum of your disc, compared to the Redump DAT if you enabled it.
+
+    ![](/images/homebrew/CleanRip/gcdumpcomplete.png)
+
+Proceed to [Managing GameCube Backups](gc-backups) to shrink the size of your output file, and manage your dumped games.
```
Comment (lifehackerhansol) [inline _pages/en_US/dump-games.md:96]: THis should also have notice--info

File: _pages/en_US/site-navigation.md [hunk 4]
```diff
@@ -53,16 +53,16 @@ sitemap: false
 {% capture notice-5 %}
 **Homebrew**
 + [Creating Wii Game Shortcuts](wiigsc)
-+ [GameCube Backup Manager](gcbackupmanager)
++ [USB Loaders](wii-loaders)
++ [Managing Wii Backups](wii-backups)
++ [Managing GameCube Backups](gc-backups)
```
Comment (lifehackerhansol) [inline _pages/en_US/site-navigation.md:58]: alphabetical order?

File: _pages/en_US/wii-backups.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
+---
+title: "Managing Wii Backups"
```
Comment (lifehackerhansol) [inline _pages/en_US/wii-backups.md:2]: As a discussion question, I wonder if we should scrap this page altogether and shove it in wiki. Dunno how I feel with it on the main guide. Dumping I can understand, not so much WBM...

(WBM is also on the wiki)

File: _pages/en_US/dump-games.md (hunk 3 above)
Comment (grunt22fox) [reply _pages/en_US/dump-games.md:96]: fixed

File: _pages/en_US/site-navigation.md (hunk 4 above)
Comment (grunt22fox) [reply _pages/en_US/site-navigation.md]: fixed the alphabetical order in the homebrew section, don't know why I changed it in the first place to be honest

File: _pages/en_US/wii-backups.md (hunk 5 above)
Comment (grunt22fox) [reply _pages/en_US/wii-backups.md:2]: this is a fair point, maybe something to discuss more in depth later

**Pull request merged**